homeassistant/components/sensor/insteon_plm.py

```diff
@@ -0,0 +1,95 @@
+"""
+Support for INSTEON dimmers via PowerLinc Modem.
```

Comment:
Please don't refactor a bunch of code _and_ add new types. THis makes code reviewing this PR horrible

Reply:
Apologies and I do appreciate your time.